app/common/state/ledgerState.js

```diff
@@ -67,12 +68,29 @@ const ledgerState = {
     return state.setIn(['ledger', 'locations', url, prop], value)
   },
 
+  getVerifiedPublisherLocationProp: (state, url) => {
+    state = validateState(state)
+    if (url == null) {
+      return null
+    }
+
+    let publisherKey = state.getIn(['ledger', 'locations', url, 'publisher'])
+
+    if (!publisherKey) {
+      const parsedUrl = urlParse(url)
+      if (parsedUrl) {
+        publisherKey = parsedUrl.hostname.replace('www', '')
```

Comment:
`parsedUrl.hostname` may be null

also this would turn `somesitewww.com` into `somesite.com`. i think you want to either omit the `replace` or use a regex (`/^www\./`)